packages/@okta/vuepress-site/docs/release-notes/2017/index.md

```diff
@@ -757,7 +769,11 @@ Version 1.13.0 of the [Okta Sign-In Widget](https://github.com/okta/okta-signin-
 
 You can configure the JIT settings for a SAML identity provider (IdP) to enable unsuspending users during inbound SAML login. See the [Identity Providers API](/docs/reference/api/idps) for more information.<!-- OKTA-128384 -->
 
-![JIT settings for SAML IdP](/img/release_notes/JIT_settings.png "JIT settings for SAML IdP")
+<div class="quarter">
+
+![JIT settings for SAML IdP](/img/release_notes/JIT_settings.png)
```

Comment:
This screenshot seems to be extremely low resolution and needs to be retaken.
Although given is a 2017 release notes, I'd suggest deleting the screenshot as it is out of date now.

Reply:
Agree; removed